package.json

```diff
@@ -235,29 +127,8 @@
 		"eslint-plugin-wpcalypso": "5.0.0",
 		"glob": "7.1.6",
 		"husky": "4.3.0",
-		"jest": "26.6.3",
```

Comment:
Since we're not installing jest anymore, we apparently need to declare the version in projects/plugins/jetpack/tests/e2e/.eslintrc.js (under settings.jest.version).
```
$ yarn lint
yarn run v1.22.10
$ eslint --ext .js,.jsx . _inc extensions modules

Oops! Something went wrong! :(

ESLint: 7.14.0

Error: Error while loading rule 'jest/no-deprecated-functions': Unable to detect Jest version - please ensure jest package is installed, or otherwise set version explicitly
Occurred while linting /usr/local/src/automattic/jetpack-reorg/projects/plugins/jetpack/tests/e2e/.eslintrc.js
```